configure.ac

```diff
@@ -352,18 +352,47 @@ AC_ARG_VAR([POLARSSL_CFLAGS], [C compiler flags for PolarSSL])
 AC_ARG_VAR([POLARSSL_LIBS], [linker flags for PolarSSL])
 if test -z "${POLARSSL_LIBS}"; then
 	AC_CHECK_LIB(
-		[polarssl],
-		[x509parse_crt],
+		[mbedtls],
+		[mbedtls_x509_crt_parse],
 		[
-			POLARSSL_LIBS="-lpolarssl"
+			POLARSSL_LIBS="-lmbedtls -lmbedx509 -lmbedcrypto"
```

Comment:
I'm not mbedtls - just a user.  As far as I know there's no pkg-config support :(